### PR TITLE
Fix same data diff format bug

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/47531

## Description
At last, Only added a beta label to IP-API and reformatted the *Location* key for **Ipstack** IP indicator.

## Does it break backward compatibility?
   - [X] Yes
       - Further details:
       the *Location* key for **Ipstack** IP indicator will now be in the format of `latitude:longitude` instead of 
      `latitude,longitude`.
   - [ ] No

### CODE REVIEW

File: Packs/IP-API/Integrations/IPAPI/IPAPI.yml [hunk 1]
```diff
@@ -34,7 +34,8 @@ description: |
   This integration will enrich IP addresses from IP-API with data about the geolocation, as well as a determination of the IP address being associated with a mobile device, hosting or proxy. Revers DNS is also returned.
 
   This service is available for free (with a throttle) - or paid.
-display: IP-API
+  This service is not following context standards, hence not recommended to use.
```
Comment (mwareman) [inline Packs/IP-API/Integrations/IPAPI/IPAPI.yml:37]: Why is this placed in my code?

File: Packs/IP-API/Integrations/IPAPI/IPAPI.yml [hunk 2]
```diff
@@ -34,7 +34,8 @@ description: |
...
+display: IP-API (beta)
```
Comment (mwareman) [inline Packs/IP-API/Integrations/IPAPI/IPAPI.yml:38]: This is NOT BETA - please revert.

File: Packs/IP-API/ReleaseNotes/1_0_2.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IP-API (beta)
+- Updated the Docker image to: *demisto/python3:3.10.4.27798*.
+- Beta. it is not following context standards, hence not recommended to use.
```
Comment (mwareman) [inline Packs/IP-API/ReleaseNotes/1_0_2.md:5]: Not BETA. Why was this added to my code?

File: Packs/IP-API/ReleaseNotes/1_0_2.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IP-API (beta)
```
Comment (mwareman) [inline Packs/IP-API/ReleaseNotes/1_0_2.md:3]: Not BETA - Please revert

**Review by mwareman — CHANGES_REQUESTED** [review]
The changes to IPSTACK do NOT require any changes to IP-API. Please revert changes commenting that the code is BETA (it's not) and not conforming to the output of another unrelated integration.

Thank you.

File: Packs/Ipstack/ReleaseNotes/1_0_6.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### ipstack
+***Breaking Change:*** Reforamtted the *Location* key for the ip indicator.
+The location key will now be in the format of `latitude:longitude` instead of `latitude,longitude`.
+- Updated the Docker image to: *demisto/python:2.7.18.27799*.
+- Added *integrationReliability* integration parameter for the DbotScore calculations.
```
Comment (ShahafBenYakir) [inline Packs/Ipstack/ReleaseNotes/1_0_6.md:7]:
```suggestion
- Added *Source Reliability* integration parameter for the DbotScore calculations.
```

**Review by ShahafBenYakir — APPROVED** [review]
Very nice! 
See my small comments

File: Tests/known_words.txt [hunk 6]
```diff
@@ -156,4 +156,8 @@ GetCampaignIncidentsInfo
 IsIncidentPartOfCampaign
 unsearchable
 ChronicleAsset
+Reforamtted
+ipstack
 SNDBOX
+C
```
[the remark this answers is not in the document]
Comment (YuvHayun) [reply Tests/known_words.txt:162]: it did

**Comment by YuvHayun** [comment]
@ShahafBenYakir  I need force merge on this PR.

**Pull request merged**